.github/workflows/ci.yml

```diff
@@ -33,6 +33,8 @@ jobs:
             **/go.sum
       - name: Build
         run: make build
+      - name: Build PF
+        run: cd pf && make build
```

Comment:
This is not strictly necessary, but I played with pulling more of the aws provider into our tests and it revealed that our lint step could time out trying to load packages for `pf/*` because we don't do a build first.